### DISCUSSION TITLE
Where can I found "QPS" and "time cost " profiling data for milvus cluster nodes? milvus v2.x

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#16034] [Q&A and General discussion]
Hi, I want to watch profiling of "QPS" and "time cost" of milvus cluster nodes in those steps : insert data, build index, search query.
But when I deploy monitor service and import json on grafana, I couldn't find them, what should I do ?

**Comment by yhmo** [comment]
https://milvus.io/docs/v2.0.x/benchmark.md#Milvus-20-Benchmark-Test-Report

**Reply by xiaojuan8** [reply]
@yhmo 抱歉，不知道是不是我表述有问题，我是想问QPS和请求查询耗时监控在哪里看？ 我grafana监控面板里没有这些数据，直接导入的milvus_dashboard.json（https://raw.githubusercontent.com/milvus-io/milvus/master/deployments/monitor/grafana/milvus-dashboard.json）
